server/grpc/orchestrator/orchestrator.proto

```diff
@@ -0,0 +1,57 @@
+syntax = "proto3";
+
+import "gogo/protobuf/gogoproto/gogo.proto";
+
+package mesg.grpc.orchestrator;
+option go_package = "github.com/mesg-foundation/engine/service/grpc/orchestrator";
+
+// This is the Orchestrator API to interact with the Orchestrator.
+// This API requires all requests to be signed for authentication. The signature should be passed in the request's metadata using the key `mesg_request_signature` and base64 encoded.
+service Orchestrator {
+  // Logs returns a stream of events that satisfy criteria specified.
+  rpc Logs(OrchestratorLogsRequest) returns (stream OrchestratorLogsResponse) {}
```

Comment:
Don't you want to have a dedicated type that can be used directly in the orchestrator and have only:
```suggestion
  rpc Logs(OrchestratorLogsRequest) returns (stream types.Log) {}
```
This is the way we already handle the events and executions

Reply:
good idea ;)